builder/bundles.go

```diff
@@ -436,17 +436,51 @@ func genUpdateBundleSpecialFiles(chrootDir string, b *Builder) error {
 	return ioutil.WriteFile(filepath.Join(swupdDir, "format"), []byte(b.State.Mix.Format), 0644)
 }
 
-func installBundleToFull(packagerCmd []string, buildVersionDir string, bundle *bundle) error {
+func downloadRpms(packagerCmd, rpmList []string, baseDir string, maxRetries int) error {
+	var downloadErr error
+
+	if maxRetries <= 0 {
```

Comment:
I believe the name is a bit confusing here since `retry` imply trying it again, but from this code, If I don't want mixer to retry a download if it fails I need to set retry to 1 instead of 0 which can be misleading. I think it would be better to check if `maxRetries >= 0` and look from `0` to `1 + maxRetries` instead, since we always wanna do at least one loop.

Reply:
Good catch! I'll update.